### PR TITLE
Prepare initial tests and class for FindMaxNumber application

### PR DESCRIPTION
**Pull request by sergtm1**
Prepare initial tests and class for FindMaxNumber application

### CODE REVIEW

File: src/main/java/com/sergtm/application/findMaxNumber/FindMaxNumber.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.sergtm.application.findMaxNumber;
+
+public class FindMaxNumber {
+    public int findMaxNum(int[][] matrix,boolean[][] availableElements){
```
Comment (aleksey-timofeev) [inline src/main/java/com/sergtm/application/findMaxNumber/FindMaxNumber.java]: Please don't forget to implement findMaxNumber function

File: src/test/java/com/sergtm/aplication/findMaxNumber/FindMaxNumberTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package com.sergtm.aplication.findMaxNumber;
+
+import junit.framework.Assert;
+
+import org.junit.Before;
```
Comment (sergtm1) [inline src/test/java/com/sergtm/aplication/findMaxNumber/FindMaxNumberTest.java]: Remove an extra import

File: src/main/java/com/sergtm/application/findMaxNumber/FindMaxNumber.java [hunk 3]
```diff
@@ -1,8 +1,16 @@
 package com.sergtm.application.findMaxNumber;
 
+
 public class FindMaxNumber {
-    public int findMaxNum(int[][] matrix,boolean[][] availableElements){
-        return 0;
-        
+    public int findMaxNum(int[][] matrix,boolean[][] availableElements,int numberOfRows,int numberOfColumns){
```
Comment (sergtm1) [inline src/main/java/com/sergtm/application/findMaxNumber/FindMaxNumber.java]: Find the way to define number of rows  and cols using matrix array

File: src/test/java/com/sergtm/aplication/findMaxNumber/FindMaxNumberTest.java [hunk 4]
```diff
@@ -21,7 +21,7 @@ public void testMatrixFiveFive() {
 
         FindMaxNumber findMaxNumber = new FindMaxNumber();
         Assert.assertEquals(
-                findMaxNumber.findMaxNum(matrix, availableElements), 8);
+                findMaxNumber.findMaxNum(matrix, availableElements, 5, 5), 8);
```
Comment (sergtm1) [inline src/test/java/com/sergtm/aplication/findMaxNumber/FindMaxNumberTest.java]: Remove number rows and columns

**Comment by sergtm1** [comment]
Please view comments and fix algorithm

File: src/main/java/com/sergtm/application/findMaxNumber/FindMaxNumber.java (hunk 1 above)
Comment (sergtm1) [inline src/main/java/com/sergtm/application/findMaxNumber/FindMaxNumber.java]: done

**Comment by aleksey-timofeev** [comment]
Looks good

**Pull request merged**